README.md

```diff
@@ -148,23 +148,21 @@ To run a functional test cluster,
 
 	cd build
 	./path/to/ceph/src/vstart.sh -d -n -x -l
```

Comment:
for the sake of consistency, could probably just use `../src/vstart.sh`? as we suggest
```
	cd build
	cmake [options] ..
	make check
```
before